code/game/machinery/suit_storage_unit.dm

```diff
@@ -201,6 +205,104 @@
 		new /obj/item/stack/sheet/metal (loc, 2)
 	qdel(src)
 
+/obj/machinery/suit_storage_unit/interact(mob/living/user)
+	var/static/list/items
+
+	if (!items)
+		items = list(
+			"suit" = create_silhouette_of(/obj/item/clothing/suit/space/eva),
+			"helmet" = create_silhouette_of(/obj/item/clothing/head/helmet/space/eva),
+			"mask" = create_silhouette_of(/obj/item/clothing/mask/breath),
+			"storage" = create_silhouette_of(/obj/item/tank/internals/oxygen),
+		)
```

Comment:
```suggestion
	var/static/list/items = list(
			"suit" = create_silhouette_of(/obj/item/clothing/suit/space/eva),
			"helmet" = create_silhouette_of(/obj/item/clothing/head/helmet/space/eva),
			"mask" = create_silhouette_of(/obj/item/clothing/mask/breath),
			"storage" = create_silhouette_of(/obj/item/tank/internals/oxygen),
		)
```
This will be assigned a single time.